GOVERNANCE.md

```diff
@@ -0,0 +1,69 @@
+# Governance
+
+## Project Roles
+
+### Contributors
+
+Individuals who want to contribute ideas to the TUF specification. Cleanups and
+clarifications are discussed in [GitHub Issues](
+https://github.com/theupdateframework/specification/issues)
+and submitted as [GitHub Pull Requests](
+https://github.com/theupdateframework/specification/pulls).
+
+New ideas and enhancements to the specification are submitted through the [TUF
+Augmentation Proposal (TAP) process](
+https://github.com/theupdateframework/taps/blob/master/tap1.md).
+
+### Consensus Builder
+
+Ultimate authority for changes to the TUF specification, including changes
+proposed through the [TAP process](
+https://github.com/theupdateframework/taps/blob/master/tap1.md), falls to the
+specification's Consensus Builder.
+
+### TAP Editors
+
+The TAP Editors are a team of core contributors to the TUF project who are
+responsible for reviewing and approving, or rejecting, any proposed
+[TAPs](https://github.com/theupdateframework/taps) and changes to the
+specification.
+
+
+## Change Review Process
+
+__All changes must be submitted as a GitHub Pull Request (PR)__
+
+The submitter of the PR is responsible for responding to feedback from
+reviewers and maintainers. While the PR remains open the submitter is also
+responsible for ensuring the change is always in a state where it can be
+merged. That is, the Pull Request does not have any unresolved conflicts,
+and has incremented the version number and last modified date as described
+in the Versioning section of the README file.
+
+__All minor changes must be approved by at least two (2) other TAP editors__
+
+Obvious language correctness (grammar and typo fixes), or other changes that
+do not significantly alter the specification must be approved by at least two
+(2) TAP Editors. These minor changes do not require a contemplation period.
+
+__All major changes must be approved by at least two (2) other TAP editors,
+and merged no sooner than one (1) week after submission__
+
+In order to ensure the security properties of TUF are maintained it is
+necessary to contemplate how any changes to the specification may affect those
+security properties. Therefore, all PRs containing non-minor changes will
+remain open for at one (1) week to allow all interested TAP Editors time to
```

Comment:
for at (least|most) one week? What is the minimum and maximum waiting time?

Reply:
Good catch, thanks!

Reply:
Fixed in 07b2b9e